man/coredump.conf.xml

```diff
@@ -119,6 +119,22 @@
         either value to 0 to turn off size-based
         clean-up.</para></listitem>
       </varlistentry>
+
+      <varlistentry>
+         <term><varname>RateLimitInterval=</varname></term>
```

Comment:
we suffix time params with `Sec` generally, to indicate the default unit. hence please rename RateLimitInterval → RateLimitIntervalSec,

Reply:
I have addressed the review comment. Please check once